dialogflow-cx/dialogflow_sample.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the base class for all Dialogflow CX samples."""
+
+
+class DialogflowSample:
+    """Base class for samples."""
+
+    def __init__(self) -> None:
+        self._agent_delegator = None
+        self._auth_delegator = None
+        self._credentials = None
+        self.test_case_delegators = None
```

Comment:
Is there a reason this attribute doesn't have the _ private prefix? It's publicly accessible?

Reply:
Missed in refactor, resolved with a public interface that protects a private implementation, same as others.